### DISCUSSION TITLE
GitHub students

### DISCUSSION THREAD

**Discussion by EllieMoynihan** [community/community#62948] [GitHub Education]
### Select Topic Area

General

### Body

Congrats! Your request for @EllieMoynihan was approved and your benefits will be available soon

I didn't know what's wrong it's been 4 day i haven't get my benefits plss help

**Comment by Akash1134** [comment]
Hi 👋  @EllieMoynihan , 

Thanks for posting in the GitHub Community. We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category [GitHub Education](https://github.com/orgs/community/discussions/categories/github-education).

I've gone ahead and moved this to the correct category for you. Good luck!

To get started, please check out [Student Developer Pack Application & FAQs ➡️](https://github.com/orgs/community/discussions/17814)
